test/e2e/operator.go

```diff
@@ -436,18 +436,20 @@ var _ = Describe("ARO Operator - ImageConfig Reconciler", func() {
 
 	// Reimplementation of function from image config controller
 	getCloudAwareRegistries := func(instance *arov1alpha1.Cluster) ([]string, error) {
-		var requiredRegistries []string
+		acrSubdomain := strings.Split(instance.Spec.ACRDomain, ".")[0]
+		var regionalRegistry string
+
 		switch instance.Spec.AZEnvironment {
 		case azureclient.PublicCloud.Environment.Name:
-			requiredRegistries = []string{instance.Spec.ACRDomain, "arosvc." + instance.Spec.Location + ".data." + azure.PublicCloud.ContainerRegistryDNSSuffix}
+			regionalRegistry = fmt.Sprintf("%s.%s.data.%s", acrSubdomain, instance.Spec.Location, azure.PublicCloud.ContainerRegistryDNSSuffix)
 
 		case azureclient.USGovernmentCloud.Environment.Name:
-			requiredRegistries = []string{instance.Spec.ACRDomain, "arosvc." + instance.Spec.Location + ".data." + azure.USGovernmentCloud.ContainerRegistryDNSSuffix}
+			regionalRegistry = fmt.Sprintf("%s.%s.data.%s", acrSubdomain, instance.Spec.Location, azure.USGovernmentCloud.ContainerRegistryDNSSuffix)
```

Comment:
Just another call for a DRY implementation of this format string (see above comments). At this point, I'm wondering if it would be good to have it as a `const`, so you can import it here for consistency.

Reply:
+1 - dry this out

Reply:
This entire function is a re-implementation of the function in image_controller.go in order to get the correct registries for the e2e test. I'd like to _not_ have to copy/paste the entire function, but making the original exportable solely for the purposes of the e2e test didn't seem appropriate. Let me know your thoughts.